pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.mdcdxcvi</groupId>
+  <artifactId>explorer</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
Leave just version 1.0, please.

Reply:
normally you use snapshot and then on every release you tag the specific version but ok, will do what you want